.github/workflows/android.yml

```diff
@@ -9,64 +9,23 @@ on:
     branches: [master, staging, next]
 
   workflow_dispatch:
+    inputs:
+      release:
+        description: 'release type'
+        required: true
+        default: 'qa'
```

Comment:
here we need just one variable - release type (dev/qa/prod)

if defined, this variable should be used instead of the current branch in all the actions below (checkout, pre checks - env is xxx, deploy to appcenter, deploy to store)

Reply:
there's an type='choice' with 'option' allowing you to create dropdowns